src/contracts/projects.ts

```diff
@@ -21,6 +21,11 @@ export type APIConfigEntity = {
   defaultValue: boolean | Record<string, any>;
   enabled: boolean;
   rules: Array<APIConfigRule>;
+  extraData: APIConfigSpecExtraData;
```

Comment:
this is already exposed ont he download config specs endpoint?   We definitely dont want to expose it to all sdks, which use the APIConfigEntity

Reply:
Not yet, I can remove it from the dcs endpoint though.